sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java

```diff
@@ -131,6 +131,8 @@ public final class SentryAndroidOptions extends SentryOptions {
    */
   private final long startupCrashDurationThresholdMillis = 2000; // 2s
 
+  private boolean enableActivityFramesTracking = true;
```

Comment:
`h`: Do we only need this for activities? I would rather go with a more generic name, if possible. Then we could also align on different platforms.
```suggestion
  private boolean enableFramesTracking = true;
```

Reply:
Right now, yes: it's only used for activity transactions and reported for slow and frozen frames. But I agree, if we e.g. want to track frame performance on a span level in future, we'll likely use the same "feature". Let me rename this.